packages/z/zeek/package.yml

```diff
@@ -0,0 +1,46 @@
+name       : zeek
+version    : 6.0.2
+release    : 1
+source     :
+    - https://github.com/zeek/zeek/releases/download/v6.0.2/zeek-6.0.2.tar.gz : 2421989adcee6a29f48a8f7272f719edbe954d66c2e86e3a52e79cae177f887c
+homepage   : https://zeek.org/
+license    : BSD-3-Clause
+component  : security
+summary    :
+    Zeek is a powerful network analysis framework that is much different from the typical IDS you may know.
+description: |
+    Zeek is not an active security device, like a firewall or intrusion prevention system. Rather, Zeek sits on a “sensor,” a hardware, software, virtual, or cloud platform that quietly and unobtrusively observes network traffic. Zeek interprets what it sees and creates compact, high-fidelity transaction logs, file content, and fully customized output, suitable for manual review on disk or in a more analyst-friendly tool like a security and information event management (SIEM) system.
+libsplit   : no
+builddeps  :
+    - pkgconfig(jemalloc)
+    - pkgconfig(krb5)
+    - pkgconfig(libcurl)
+    - pkgconfig(libmaxminddb)
+    - pkgconfig(libpcap)
+    - pkgconfig(python3)
+    - flex-devel
+    - swig
+setup      : |
+    %cmake_ninja . \
+    -D ZEEK_ETC_INSTALL_DIR=/usr/share/defaults/etc/zeek \
+    -D ZEEK_LOCAL_STATE_DIR=/var/lib/zeek \
+    -D ZEEK_SPOOL_DIR=/var/log/zeek/spool \
+    -D ZEEK_LOG_DIR=/var/log/zeek/logs \
+    -D ENABLE_JEMALLOC=ON \
+    -D INSTALL_BTEST=ON \
+    -D INSTALL_BTEST_PCAPS=ON \
+    -D INSTALL_ZEEK_ARCHIVER=ON \
+    -D INSTALL_ZEEK_CLIENT=ON \
+    -D INSTALL_AUX_TOOLS=ON \
+    -D INSTALL_ZEEKCTL=ON \
+    -D INSTALL_ZKG=OFF
+build      : |
+    %ninja_build
+patterns   :
+    - /*
+install    : |
+    %ninja_install
+replaces   :
+    - bro
+    - devel   : bro-devel
```

Comment:
If you're not creating a `-devel` subpackage then the replaces of the `bro-devel` package should be attached to the main package.